python/sglang/srt/mem_cache/multimodal_cache.py

```diff
@@ -100,11 +104,18 @@ def set(
             self.mm_cache.move_to_end(mm_hash)
             return True
         data_size = _get_tensor_size(embedding)
+        evicted_bytes = 0
         while self.current_size + data_size > self.max_size:
             if not self.mm_cache:
                 return False
             lru_hash, lru_embedding = self.mm_cache.popitem(last=False)
-            self.current_size -= _get_tensor_size(lru_embedding)
+            freed = _get_tensor_size(lru_embedding)
```

Comment:
This comment doesn't belong to this line. It seems mm_embedding_allocator it has never been used.

Reply:
Got it. Looks good.